llvm/lib/Target/VE/MCTargetDesc/VEInstPrinter.cpp

```diff
@@ -66,9 +66,9 @@ void VEInstPrinter::printOperand(const MCInst *MI, int OpNum,
 
 void VEInstPrinter::printMemASXOperand(const MCInst *MI, int OpNum,
                                        const MCSubtargetInfo &STI,
-                                       raw_ostream &O, const char *Modifier) {
+                                       raw_ostream &O, StringRef Modifier) {
```

Comment:
VE doesn't use modifiers, so this (and below) should be dropped too. I guess this was copied from Sparc, where I removed it some time ago.

Reply:
Thanks, I guess I am not sure how to tell it immediately, but I removed it from the VE side.